### PR TITLE
kind of cleanup

### PR DESCRIPTION
**Pull request by HuKeping**
Signed-off-by: Hu Keping hukeping@huawei.com

### CODE REVIEW

**Comment by coolljt0725** [comment]
LGTM

File: registry/registry.go [hunk 1]
```diff
@@ -237,8 +237,7 @@ func ContinueOnError(err error) bool {
 // default TLS configuration.
 func NewTransport(tlsConfig *tls.Config) *http.Transport {
 	if tlsConfig == nil {
-		var cfg = tlsconfig.ServerDefault
-		tlsConfig = &cfg
+		tlsConfig = &tlsconfig.ServerDefault
```
Comment (duglin) [inline registry/registry.go]: Need someone to double check but I don't think these are the same things. I think the original will duplicate ServerDefault, while the new one just points to the default. Which means if someone (later on) modifies "tlsConfig" then they'll end up changing the default, which would be bad.

Comment (duglin) [inline registry/registry.go]: http://play.golang.org/p/oBmvEMovaW

Comment (vdemeester) [inline registry/registry.go]: @duglin yep, I do agree.

**Comment by duglin** [comment]
@HuKeping you ok with closing this?

**Comment by calavera** [comment]
I'm closing this, since it introduces a bug. Thanks for checking @duglin.

File: registry/registry.go (hunk 1 above)
Comment (HuKeping) [inline registry/registry.go]: Ah, that's a mistake indeed, agreed.

**Comment by coolljt0725** [comment]
@duglin  Good point :+1: ,  sorry for my bad review :sweat_smile:

**Pull request closed without merging**